.gitattributes

```diff
@@ -64,7 +64,7 @@
 
 # Don't allow people to merge changes to these generated files, because the result
 # may be invalid.  You need to run "rush update" again.
-pnpm-lock.yaml               merge=binary
+pnpm-lock.yaml               merge=text
```

Comment:
This is generated from the `rush init` template.  If the best practice has changed, we should reflect that in the template

Reply:
Also this new line is inconsistent with the `#` comment above it

Reply:
😂 Didn't realize this was the TSDoc repo -- I thought it was the Rush Stack repo.

Reply:
This is from `rush init`.